training-maquetado/scss/detail.scss

```diff
@@ -1,9 +1,11 @@
 @import './variables/_sizes';
 
 .book {
+  box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
   display: flex;
   height: 416px;
   padding: 24px;
+  width: 1000px;
```

Comment:
we try to not set a fix width so it is adaptable to diferent screen sizes. Use this:
```
width: 100%
max-width: 1000px
```
it will have the same result and in the case the screen is less than `1000px` it will be able to accommodate to the correct size

Reply:
This is what i was looking for and i couldn't find it, tysm.